### ISSUE TITLE
Invalid ACIdentifier when using rkt runtime

### ISSUE THREAD

**Issue by shanegibbs** [kubernetes/kubernetes#20724]
I have built kubernetes master (386aab82deefdce5c81fd921880070bfe1ffebf7) and am using rkt 1.0.0 as the runtime. I am encountering the following error in the kubelet when trying to start a pod:

```
E0205 09:20:30.266217    2856 runtime.go:58] Recovered from panic: "ACIdentifier must contain only lower case alphanumeric characters plus \"-._~/\"" (ACIdentifier must contain only lower case alphanumeric characters plus "-._~/")
```

**Comment by shanegibbs** [comment]
After digging in the code, it seems that something is trying to use an invalid ACI identifier: `rkt.kubernetes.io/terminationMessagePath`, [here](https://github.com/kubernetes/kubernetes/blob/386aab82deefdce5c81fd921880070bfe1ffebf7/pkg/kubelet/rkt/rkt.go#L621). Which, according to the panic, is invalid due to having uppercase chars.

After looking over the ACI spec, the regex in  [acidentifier.go](https://github.com/appc/spec/blob/3972f6a1b657f38e30100d2a7ef377cc8690c8db/schema/types/acidentifier.go#L25-L29) would also confirm this ACI in invalid.

**Comment by shanegibbs** [comment]
Downcasing seems to work - https://github.com/kubernetes/kubernetes/pull/20728. However, I do not fully understand the ramifications of this change.

**Comment by shanegibbs** [comment]
Looks like something @yifan-gu may know about...

**Comment by yujuhong** [comment]
@shanegibbs, thanks for reporting. The fix #20816 is to merged soon.
